goplay/src/main/java/be/tapped/goplay/model.kt

```diff
@@ -39,6 +44,13 @@ public sealed interface ApiResponse {
             public object NoPrograms : Content
         }
 
+        public sealed interface Stream : Failure {
+            public data class MpegDash(val json: JsonObject, val throwable: Throwable) : Stream
```

Comment:
Maybe I might condense this to 1 type. Since we can deduce from the JSON what went wrong

Reply:
Hmm or maybe not.